packages/url_launcher/url_launcher_platform_interface/lib/url_launcher_platform_interface.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+/// The interface that implementations of url_launcher must implement.
+abstract class UrlLauncherPlatform {
```

Comment:
Add to the class doc that implementers should extend this class and not implement it, otherwise we won't be able to add new methods.

See similar comment on webview: https://github.com/flutter/plugins/blob/fe9b5594f92fceafe2031c20aba586eb350e00ea/packages/webview_flutter/lib/platform_interface.dart#L35

Reply:
Done.